js/src/lib/components/DropdownMenu/DropdownMenu.svelte

```diff
@@ -47,7 +47,7 @@
 	class="absolute top-full mt-1 min-w-full w-auto bg-white rounded-xl overflow-hidden shadow-lg z-10 border border-gray-100
 		{alignement === 'right' ? 'right-0' : 'left-0'}
 		{classNames}"
-	on:click={onClose}
+	on:click|stopPropagation={onClose}
```

Comment:
This contributes to fix the issue of Dropdown menus not closing properly when the Dropdown is inside a Modal window.